x-pack/plugins/canvas/public/components/workpad_static_page/static_workpad_page.js

```diff
@@ -6,7 +6,7 @@
 
 import React, { PureComponent } from 'react';
 import { ElementWrapper } from '../element_wrapper';
-import { staticWorkpadPagePropTypes } from './prop_types';
+import { staticWorkpadPagePropTypes } from '../workpad_page/prop_types';
```

Comment:
nit: I didn't love the proptype separation before, and I think it makes even less sense now...

Reusing `staticWorkpadPagePropTypes` seems appealing, but it also makes an already intimidating chunk of code harder to understand since you have to go track down the props in another file.

Reply:
I had the same feeling and swept it away as the plan is to remove these runtime prop defs and it felt somewhat better than duplicating them - should I do the duplication though? Would be a quick thing.

Reply:
> should I do the duplication though? 

On the surface, I would say yes, but if you have future plans already, maybe it makes sense to wait. Up to you.